master/getting-started/kubernetes/installation/hosted/k8s-backend/index.md

```diff
@@ -18,6 +18,7 @@ You must have a cluster which meets the following requirements:
 
 - You have a Kubernetes cluster configured to use CNI network plugins (i.e by passing --network-plugin=cni to the kubelet)
 - Your Kubernetes controller manager is configured to allocate pod CIDRs (i.e by passing --allocate-node-cidrs=true to the controller manager)
+- Your Kubernetes controller manager has been provided a cluster-cidr (the manifest expects --cluster-cidr=10.244.0.0/16 by default).
```

Comment:
I kind of want the `(the manifest....)` to use the pattern of `(i.e by passing...)`.  Maybe something like `(i.e. by passing --cluster-cider=10.244.0.0/16, which the manifest expects by default)`